okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -233,8 +233,8 @@ public void setProxyService(ProxyService px) {
       mods.put(md.getId(), md);
     }
     if (modTo == null) {
-      String deps = DepResolution.checkAllDependencies(mods);
-      if (!deps.isEmpty()) {
+      List<String> errors = DepResolution.checkEnabled(mods);
+      if (!errors.isEmpty()) {
         return Future.succeededFuture(); // failures even before we remove a module
```

Comment:
You haven't changed this, but why does this return success suppressing both any existing and any new resolution errors?

Reply:
The problem is that if a set of modules are already inconsistent, then we allow it to proceed anyway.

Reply:
Can we have a warning in the log?

Reply:
Sure